dapr/clients/http/dapr_invocation_http_client.py

```diff
@@ -39,7 +39,8 @@ def invoke_method(
             content_type: Optional[str] = None,
             metadata: Optional[MetadataTuple] = None,
             http_verb: Optional[str] = None,
-            http_querystring: Optional[MetadataTuple] = None) -> InvokeMethodResponse:
```

Comment:
How about this? Can you just add an invoke_method_async call? I think having separate synchronous and async signatures might avoid confusion and maintain backwards compatibility.

Reply:
Thanks for the response. This does indeed sound like a better approach.
Can you review this updated PR?